web/packages/teleterm/src/ui/ClusterConnect/ClusterLogin/useClusterLogin.ts

```diff
@@ -292,6 +292,11 @@ async function loginSso(
     },
     { abort: abortSignal }
   );
+
+  // Force once login finishes but before we await the cluster sync. This way the focus will go back
+  // to the app ASAP. The login modal will be shown until the cluster sync finishes.
+  void mainProcessClient.forceFocusWindow();
```

Comment:
I guess it'd be nice to force focus after an error as well. However, we'd need to differentiate between a timeout (that's set deep in `lib/client/sso/redirector.go`) and a regular error so that we don't steal focus on a timeout.

As it involves passing the error through multiple layers (lib/client -> lib/teleterm -> the Electron app), it might be a bit more complex and I didn't have the time to address it during today's support shift.